tools/cli/commands/changelog.js

```diff
@@ -213,6 +214,26 @@ export function changelogDefine( yargs ) {
 					async argv => {
 						await changelogArgs( argv );
 					}
+				)
+				// Squash subcommand.
+				.command(
+					'squash [project]',
+					'Squashes changelog projects',
+					yargSquash => {
+						yargSquash
+							.positional( 'project', {
+								describe: 'Project in the form of type/name, e.g. plugins/jetpack',
+								type: 'string',
+							} )
+							.option( 'readme', {
+								describe: 'Indicates if we only want to squash the readme and not the changelog.',
+								alias: 'r',
+								type: 'bool',
```

Comment:
"bool" isn't actually a yargs type.
```suggestion
								type: 'boolean',
```
Looks like lots of other args in this file have the same mistake. There's also one in tools/cli/commands/docker.js.

Reply:
Thanks for noticing, I'll clean those up 👍